source/docs/05-mensural.xml

```diff
@@ -382,7 +382,7 @@
                         <specDesc key="att.xy" atts="x y"/>
                      </specList>
                   </p>
-                  <p>PLACE-HOLDER FOR EXAMPLE OF A NOTE WITH TWO STEMS: IMAGE &amp; ITS ENCODING</p>
+                  <p>[include example (image and code) of a note with two stems]</p>
```

Comment:
Is there already such an example? Otherwise we probably should mark this as XML-comment and add `<!--- TODO -->`before it?